src/transformStateWithClones.js

```diff
@@ -6,8 +6,35 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const res = [];
+  const clone = { ...state };
+
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
`for (const action of actions)`

Reply:
+ object destruction

Reply:
I don't understand how I should apply destructuring here and what exactly I should specify in default.
Help me please